recipe/meta.yaml

```diff
@@ -11,18 +11,12 @@ source:
 
 build:
   number: 1
-  msvc_compiler: 14.0  # [win]
```

Comment:
Initially was going to leave this in with the `{{ compiler("cxx") }}` syntax below. However it turns out that doesn't seem to work. Could be an error on my part or it could be a bug. ( https://github.com/conda/conda-build/issues/2866 )

Anyways dropping this and using the `skip` below worked great. So went with that. :)